test/integration/sagemaker/test_mnist.py

```diff
@@ -83,6 +85,22 @@ def test_distributed_mnist_ps(sagemaker_session, ecr_image, instance_type):
     _assert_s3_file_exists(os.path.join(estimator.model_dir, 'model.ckpt-0.meta'))
 
 
+def test_s3_plugin(sagemaker_session, ecr_image, instance_type):
+    resource_path = os.path.join(os.path.dirname(__file__), '..', '..', 'resources')
+    script = os.path.join(resource_path, 'mnist', 'checkpoint_every_step_mnist.py')
+    estimator = TensorFlow(entry_point=script,
+                           role='SageMakerRole',
+                           train_instance_count=1,
+                           train_instance_type=instance_type,
+                           sagemaker_session=sagemaker_session,
+                           image_name=ecr_image,
+                           framework_version='1.12.0',
+                           py_version='py3',
+                           base_job_name='test-tf-sm-distributed-mnist')
+    estimator.fit(MNIST_DATA_S3_PATH)
+    _assert_s3_file_exists(estimator.model_data)
```

Comment:
can you also add an assert that's specific to the checkpointing part?

Reply:
yeah. will do